shared/chat/inbox/container/__tests__/container.test.js

```diff
@@ -14,27 +14,29 @@ describe('inbox container tests', () => {
     expect(withYou).toBe(withoutYou)
   })
   it('searching for nothing', () => {
-    expect(_testing.score('nothing', 'myname', ['a', 'b', 'c'])).toBe(0)
+    expect(_testing.score('nothing', 'myname', ['a', 'b', 'c'])).toBeLessThanOrEqual(0)
   })
   it('searching for exact', () => {
-    const oneExactScore = 100 / 3
-    expect(_testing.score('exact', 'myname', ['exact', 'b', 'c'])).toBeCloseTo(oneExactScore)
-    expect(_testing.score('exact', 'myname', ['a', 'exact', 'c'])).toBeCloseTo(oneExactScore)
-    expect(_testing.score('exact', 'myname', ['a', 'b', 'exact'])).toBeCloseTo(oneExactScore)
+    const s1 = _testing.score('exact', 'myname', ['exact', 'b', 'c'])
+    const s2 = _testing.score('exact', 'myname', ['a', 'exact', 'c'])
+    const s3 = _testing.score('exact', 'myname', ['a', 'b', 'exact'])
+    expect(s1).toBe(s2)
+    expect(s2).toBe(s3)
   })
+  // This isn't strictly true as there's some points per number of participants
```

Comment:
outdated comment for the current version, since you just care whether there's an exact/prefix/substr match or not?

Reply:
this relationship is mostly true but it can be violated if the number of participants is large. it largely captures what we're trying to do